R/repair-column-names.R

```diff
@@ -0,0 +1,36 @@
+#' Check validity/format of a data.frame or tbl.
+#'
+#' Ensure the tbl or data.frame has legitimate and optionally unique
+#' column names.
+#' @param x tbl, tbl_df, or data.frame
+#' @param prefix character, the prefix to use for new colum names
+#' @param sep character, a character string to separate the terms. Not
+#'     \code{NA_character_}.
+#' @return data.frame with possibly-repaired column names
+#' @export
+repair_names <- function(x, prefix = "V", sep = "") {
+  if (! isTRUE(ncol(x) > 0)) return()
```

Comment:
Should return x in this case; I think we still should throw an error if x isn't an object that can have column names.

Should this function also work for lists?